grml-debootstrap

```diff
@@ -1687,30 +1696,20 @@ iface eth0 inet dhcp
 "
 
   if [ -n "$NOINTERFACES" ] ; then
-    einfo "Not installing /etc/network/interfaces as requested via --nointerfaces option" ; eend 0
+    einfo "Not installing /etc/systemd/network/80-dhcp.network as requested via --nointerfaces option"
+    eend 0
   elif [ -n "$USE_DEFAULT_INTERFACES" ] ; then
-    einfo "Installing default /etc/network/interfaces as requested via --defaultinterfaces options."
+    einfo "Installing default /etc/systemd/network/80-dhcp.network as requested via --defaultinterfaces options."
+    echo "$DEFAULT_SYSTEMD_NETWORKD" > "${MNTPOINT}/etc/systemd/network/80-dhcp.network"
     echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
     eend $?
   elif [ -n "$VIRTUAL" ] ; then
-    einfo "Setting up Virtual Machine, installing default /etc/network/interfaces"
-    echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
-    eend $?
-  elif [ -r /etc/network/interfaces ] ; then
-    einfo "Copying /etc/network/interfaces from host to target system"
-    cp $VERBOSE /etc/network/interfaces "${MNTPOINT}/etc/network/interfaces"
-    eend $?
-  else
-    ewarn "Couldn't read /etc/network/interfaces, installing default /etc/network/interfaces"
+    einfo "Setting up Virtual Machine, installing default /etc/systemd/network/80-dhcp.network"
+    echo "$DEFAULT_SYSTEMD_NETWORKD" > "${MNTPOINT}/etc/systemd/80-dhcp.network"
```

Comment:
This should be /etc/systemd/network/80-dhcp.network?

Reply:
Hmm.. There was a hickup in my rebase :) This should have already been fixed. Will update the PR.